src/logic/Windows.swift

```diff
@@ -8,7 +22,7 @@ class Windows {
 
     // reordered list based on preferences, keeping the original index
     static func reorderList() {
-        list.sort {
+        list = list.stableSorted {
```

Comment:
Could you share why this was needed?

Reply:
My thinking at the time was that we would prefer to have the windows consistently stay in the same order and that without a stable sort windows with the same name could arbitrarily change positions.  Given that I fallback to the recency ordering when they have the same title, there is no need for this...  Thoughts?

Reply:
Yes indeed, it's already deterministic without this

Reply:
To be clear. from my reading list.sort is not stable, so in the event that the result of a comparison is equal, then the order is non-deterministic.  That said, your focus Id seems to make this deterministic.